src/components/LibraryContainer.tsx

```diff
@@ -293,6 +303,7 @@ export class LibraryContainer extends React.Component<LibraryContainerProps, Lib
 
     updateSearchViewDelayed(text: string) {
         if (text.length == 0) {
+            LibraryUtilities.setItemStateRecursive(this.generatedSections, true, false);
```

Comment:
If you search in the code when the updateSearchViewDelayed() is called the LibraryUtilities.setItemStateRecursive() method was already called then why call this method again?

Reply:
yeah, I needed to do this when I had reverted that `onWillRecieveProps` hook where the props override state, so expanded was not getting reset - I will see if I can remove this now.

Reply:
this is still necessary, though I moved it out of the `updateSearchViewDelayed` method - it's just a bug that only exists in the librarieJS test app, not in the dynamo context - see the comments I added in latest commit.